### ISSUE TITLE
OCPP Server uses to IPV6 on Ubuntu

### ISSUE THREAD

**Issue by Simian60** [Argonne-National-Laboratory/node-red-contrib-ocpp#62]
It would be nice to have a way of specifying the address to listen on as well as the port in the node-red UI.

Hi, 

As title, running on Ubuntu the OCPP server listens on [::]:port by default, rather than 0.0.0.0:port. I think this is because 'localhost' gets an ipv6 address rather than an ipv4 one.

Obviously my car charger doesn't have a clue about ipv6, and won't connect using it.

**Comment by bnystrom** [comment]
Do you require ipV6 on your Ubuntu server? If not, you should be able to disable it and have it only pick up ipV4.

**Comment by Simian60** [comment]
> Do you require ipV6 on your Ubuntu server? If not, you should be able to disable it and have it only pick up ipV4.

Yeah, that's what I'm going to try this weekend. I don't think anything else relies on it.

> And depending on your local router's abilities to support local DHCP, you might be able to connect to the server using its hostname.

For reasons I can't work out, only wired not wifi connected machines are available via the routers DNS. It's ISP provided, so there's no hope of anything being fixed on it. The ubuntu box's main job is controlling the house solar / batteries over RS485, and running a network cable to it would involve more drilling through external walls than I want to do.

Cheers,

Simon.
